src/covid_shared/cli_tools.py

```diff
@@ -42,6 +42,16 @@ def configure_logging_to_terminal(verbose: int):
     add_logging_sink(sys.stdout, verbose, colorize=True)
 
 
+def make_log_path(output_path: Path) -> None:
```

Comment:
Is there a good reason to keep this separate from the `configure_logging_to_files`?

Reply:
I suppose not. I didn't combine because In other packages I had headaches dealing with a `configure_logging`-esque function getting imported and called in many modules. But with a cli this should be done just one per process so I don't see any downsides to having side effects inside `configure_logging_to_files`. Will switch